src/native/libs/System.Native/pal_datetime.m

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "pal_datetime.h"
+#import <Foundation/Foundation.h>
+
+char* SystemNative_GetDefaultTimeZone()
+{
+    NSTimeZone *tz = [NSTimeZone localTimeZone];
```

Comment:
I'm noticing there are `system`, `local`, and `'default'` TimeZone variables in the `Working with System Time Zones` section of https://developer.apple.com/documentation/foundation/nstimezone. Do we want to use the cached system's time zone itself, an object reflecting the system's time zone even if it changes, or the time zone used by apps (that can be changed by the app itself?)

Tracing the purpose of `TryGetLocalTzFile`, it looks like this is called when instantiating the cached data and seemingly nowhere else. The behavior of the cached data seems to be similar to `systemTImeZone` https://developer.apple.com/documentation/foundation/nstimezone/1387231-system, but not sure if theres a difference for our usage.

Reply:
`localTimeZone` was the previous behavior in Xamarin iOS.